src/group_theory/group_action/defs.lean

```diff
@@ -249,6 +267,23 @@ a multiplicative action of `N` on `α`. -/
 an additive action of `N` on `α`. -/
 add_decl_doc add_action.comp_hom
 
+variable {α}
+
+/-- The tautological action by `function.End α` on `α`. -/
+instance function_End : mul_action (function.End α) α :=
+{ smul := ($),
+  one_smul := λ _, rfl,
+  mul_smul := λ _ _ _, rfl }
+
+/-- The monoid hom representing a monoid action. -/
```

Comment:
```suggestion
/-- The monoid hom representing a monoid action.

When `M` is a group, see `mul_action.to_perm_hom` -/
```

Can you add the additive versions of these too, to match `add_action.to_perm_hom`?

Reply:
Sure